WebContent/active/Unvalidated-Redirect/Redirect-FalsePositives-GET/inclusion-logic3.jsp

```diff
@@ -282,9 +282,17 @@
                     	}
                     					
                     	if (defaultInputType == DefaultInputType.RELATIVE_INPUT) {
+                            // Use JMX to get the hosts from the Tomcat (specific) config
+                            // There should only be one configured (default is localhost)
```

Comment:
This is explaining what's doing (which one can tell by reading the code) not the why which is what we should tell (without having to go to the changelog).